src/vscode-dts/vscode.proposed.interactive.d.ts

```diff
@@ -93,7 +93,7 @@ declare module 'vscode' {
 	export interface InteractiveSession {
 		requester: InteractiveSessionParticipantInformation;
 		responder: InteractiveSessionParticipantInformation;
-		inputPlaceholder?: string;
+		inputValue?: string;
```

Comment:
Is this property in use somewhere? We should be careful not to break existing extension consumers